packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -901,14 +901,14 @@ class _DatePickerHeader extends StatelessWidget {
 /// returned.
 ///
 /// If [initialDateRange] is non-null, then it will be used as the initially
```

Comment:
References to the parameter also be using backquotes, since it's not a member, but a parameter.

```dart
/// If `initialDateRange` is non-null, then it will be used as the initially
```